app/css/base.css

```diff
@@ -21,3 +21,6 @@ h2 {
 .vbox {
   min-height: 0;
 }
+.text-blue {
```

Comment:
I changed my mind because it will also be used for dates, so it's a nice kind of "variable". Let's keep it.